### ISSUE TITLE
The amount of data is increasing

### ISSUE THREAD

**Issue by KyryloKrasiuk** [bbernhard/signal-cli-rest-api#494]
### The problem

The volume of data in storage is constantly increasing. There were big jumps in the native mode (it can be seen on the graph). Now, in json-rpc mode, space is not being used so quickly.

What takes up so much space? I can't look directly into the container because railway.app doesn't have this functionality.
Is it possible to add some cleanup of unnecessary data?

![image](https://github.com/bbernhard/signal-cli-rest-api/assets/2129359/f14bfe57-f2f5-468c-87ce-19b222263a02)

### Are you using the latest released version?

- [X] Yes

### Have you read the troubleshooting page?

- [X] Yes

### What type of installation are you running?

signal-cli-rest-api Docker Container

### In which mode are you using the docker container?

JSON-RPC Mode

### What's the architecture of your host system?

arm64

**Comment by bbernhard** [comment]
Haven't seen this before. Without additional info I guess it is also quite hard to track down the reason for that.

**Comment by KyryloKrasiuk** [comment]
I think it's the data in the `attachments` directory. Should they be kept or deleted?

**Comment by bbernhard** [comment]
Did you send the attachments via the REST API oder did you receive them?

**Comment by KyryloKrasiuk** [comment]
This is from the received.
I see .png, .mp3, and .mp4 files. I think it's from different contacts and groups.

**Comment by bbernhard** [comment]
There's no cleanup for that, but you can remove it manually.

**Comment by KyryloKrasiuk** [comment]
Yes, I was thinking of creating a cron command to delete, but if the service does not save messages, then maybe the files should be cleaned?

**Comment by KyryloKrasiuk** [comment]
Understood. Thank you.
